public/docs/ts/latest/guide/security.jade

```diff
@@ -118,47 +116,42 @@ figure.image-display
-  ### Use the Offline Template Compiler
+  ### Use the offline template compiler
 
   The offline template compiler prevents a whole class of vulnerabilities called template injection,
   and also greatly improves application performance. Use the offline template compiler in production
-  deployments. Do not dynamically generate templates. Angular trusts template code, so generating
-  templates, in particular containing user data, circumvents Angular's built-in protections. See the
-  [Dynamic Forms Cookbook](../cookbook/dynamic-form.html) on how to dynamically construct forms in a
-  safe way.
+  deployments; do not dynamically generate templates. Angular trusts template code, so generating
+  templates, in particular templates containing user data, circumvents Angular's built-in protections. For information about how to dynamically construct forms in a safe way, see 
+  [Dynamic Forms Cookbook](../cookbook/dynamic-form.html).
 
-  ### Server side XSS protection
+  ### Server-side XSS protection
 
   HTML constructed on the server is vulnerable to injection attacks. Injecting template code into an
   Angular application is the same as injecting executable code into the
-  application; it gives the attacker full control over the application. To prevent this, make sure
-  to use a templating language that automatically escapes values to prevent XSS vulnerabilities on
-  the server. Do not generate Angular templates on the server side using a templating language, this
-  carries a high risk of introducing template injection vulnerabilities.
+  application: it gives the attacker full control over the application. To prevent this, 
+  use a templating language that automatically escapes values to prevent XSS vulnerabilities on
+  the server. Do not generate Angular templates on the server side using a templating language; doing this
+  carries a high risk of introducing template-injection vulnerabilities.
 
 .l-main-section
-h2#bypass-security-apis Trusting Safe Values
+h2#bypass-security-apis Trusting safe values
 :marked
   Sometimes applications genuinely need to include executable code, display an `<iframe>` from some
-  URL, or construct potentially dangerous URLs. To prevent automatic sanitization in this situation,
-  you can tell Angular that you inspected a value, checked how it is generated, and made sure it is
-  always secure. But **be careful**! If you trust a value that can be malicious, you will likely
-  introduce a security vulnerability into your application. If in doubt, find a professional
-  security reviewer.
+  URL, or construct potentially dangerous URLs. To prevent automatic sanitization in any of these situations, you can tell Angular that you inspected a value, checked how it was generated, and made sure it will always be secure. But **be careful**! If you trust a value that might be malicious, you are introducing a security vulnerability into your application. If in doubt, find a professional security reviewer.
```

Comment:
This is a verrrry long line... I like to keep lines <=80 chars, so it's easier to see diffs in GitHub and other tools.